configure-buildfarm/defaults/main.yml

```diff
@@ -1,7 +1,6 @@
 ---
 
 project_name: digger
-buildfarm_version: "2.0"
```

Comment:
is it ok to remove this yeah?

Reply:
this is the only reference to the variable in this installation. For rhmap this value is populated by the vars/buildfarm.yml. See references [here](https://github.com/fheng/rhmap-ansible/search?utf8=%E2%9C%93&q=buildfarm_version&type=)